Chapter-4-applying-tactical-domain-driven-design/Fitnet.Contracts/Src/Fitnet.Contracts.Application/AttachAnnexToBindingContract/AttachAnnexToBindingContractCommandHandler.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace EvolutionaryArchitecture.Fitnet.Contracts.Application.AttachAnnexToBindingContract;
+
+using EvolutionaryArchitecture.Fitnet.Common.Api.ErrorHandling;
+
+[UsedImplicitly]
+internal sealed class AttachAnnexToBindingContractCommandHandler(
```

Comment:
What you think about returing annex id. To build better rest API. We can do it in next MR